docs/install-and-run.md

```diff
@@ -3,115 +3,126 @@
+#### Step 1: Set up a Unit target      ///// EarlGrey adds changes to the test target's Scheme and Build Phases
 
-#### Step 1: Set up a test target
+1. To create a new Unit Test (XCTest) target, select your project in the Xcode Project Navigator,
+and then click **Editor → Add Target...** from the menu.
+2. In the **Add Target** window, select **iOS** → **Test** → **iOS Unit Testing Bundle**:
 
-  1. EarlGrey requires a **Test Target**. Because EarlGrey adds changes to the test target's Scheme and Build
-  Phases, we recommend that you create a separate test target for adding EarlGrey tests. If you do not have
-  one set up already, you can do so by selecting your project in the Xcode Project Navigator, and then and
-  clicking **Editor → Add Target...** from the menu.
-  2. In the **Add Target** window, select **iOS** → **Test** → **iOS Unit Testing Bundle**:
+<img src="images/image00.png" width="500">
 
-      <img src="images/image00.png" width="500">
+Click Next → "Add a Test Target Name" → Finish.
 
-  3. Because EarlGrey uses Schemes, the test target must have a Scheme associated with it. It is also
-  better if the Scheme is shared. If your test target does not have a Scheme, go to **Product** → **Scheme** → **Manage Schemes**, press
-  the plus **(+)** sign, and then select the target from the dropdown menu. Select the **Shared** option and
-  have the **Container** set to the app under test.
+3. The test target must have a Scheme associated with it. To add one, go to **Product** →
+**Scheme** → **Manage Schemes**, press the plus **(+)** sign, and then select the target from
+the dropdown menu. Ensure the **Container** is set to the app under test.
 
-      <img src="images/image01.png" width="500">
+<img src="images/image01.png" width="500">
 
-     Note: If you recently created the Schemes, you need to run them once for them to be picked up by the
-     `pod install` command. If your **Test Target(s)** do not contain these changes to the
-     [Scheme](#scheme-changes) and [Build Phases](#build-phase-changes) after running
-     `pod install`, please re-run `pod install` again.
+And clock on Close.
 
 #### Step 2: Add EarlGrey as a framework dependency
 
-  1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
-  2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
-     in your `post_install` actions to add Build Settings and Scheme changes to the **Test Target**
-     for embedding EarlGrey in your app. To use the setup gem, use `gem install earlgrey` before
-     `pod install`.
+1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
+2. Add EarlGrey as a dependency on your Test Target as in below.
+
+```
+target TEST_TARGET do
+project PROJECT_NAME
+
+use_frameworks! # Required for Swift Test Targets only
+inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+pod 'EarlGrey'
```

Comment:
This is already in a formatted block. Adding **...** doesn't make the text bold here.